dbt/node_runners.py

```diff
@@ -250,14 +250,6 @@ def call_already_exists(schema, table):
             "already_exists": call_already_exists,
         }
 
-    @classmethod
```

Comment:
I found this dead code when I was trying to track down who called `get_existing_schemas` and what cared about what aspects, so I removed it as part of this rather than spend any mental energy on upkeep.